app/views/companies/show.html.haml

```diff
@@ -42,9 +42,9 @@
         = field_or_none t('.website'), link_to(@company.website, sanitize(full_uri(@company)), target: '_blank'), tag_options: {class: 'website'}
 
         .social-media
-          = icon_link(@company.facebook_url, 'facebook')
-          = icon_link(@company.instagram_url, 'instagram')
-          = icon_link(@company.youtube_url, 'youtube-square')
+          = icon_link(@company.facebook_url, 'facebook', 'Facebook URL', @company.facebook_url)
+          = icon_link(@company.instagram_url, 'instagram', 'Instagram URL', @company.instagram_url)
+          = icon_link(@company.youtube_url, 'youtube-square', 'Facebook URL', @company.youtube_url)
 
```

Comment:
Would it be more user-friendly to have alt be 'Link to Facebook', rather than 'Facebook URL'?

Similarly, title might be 'Link to <company> Facebook page' rather than a URL.

Reply:
@patmbolger Just changed the PR according to your review, my apologies for this huge delay.

Reply:
@chethanagopinath No problem with the delay ... we're all busy with life.  Anyway, the changes look good to me.

Now, you'll have to localize the text strings you are using (I probably should have mentioned this before).  That is, leverage Rails i18N API.  We need to support Swedish (for users) and English (as an aid to developers).

I'd start with @weedySeaDragon 's write-up in the wiki: https://github.com/AgileVentures/shf-project/wiki/How-to-Use-I18n-to-Localize-SHF.   Then, of course, review the Rails guide (linked to in wiki page).  You'll find plenty of examples in the actual translation files.  (I know this seems like a lot of work for a small change, but it's actually not, and knowing how i18N works is a valuable skill.)

Reply:
@patmbolger Hi, I just wrapped up my semester and am looking to complete the PR. I hope I can still work on it and the issue still remains. 

Within `/locals/en.yml`, there is a key `company.facebook_url` which has the value `Facebook page URL`. I created my own keys for `alt` and `title` for Facebook with `company.facebook_url_alt` and `company.facebook_url_title` respectively and am looking to do the same for Instagram and YouTube. I just wanted to check with you if I need to remove the pre-existing key `company.facebook_url` or I should reuse that for my needs. 

Also, I should translate the English text to Swedish and add it in `/locals/sv.yml` right? I was concerned as I see a little bit of English in the Swedish file and vice versa.

Reply:
@chethanagopinath Welcome back - I hope the semester went well.  

Please leave that `company.facebook_url` key in place.  I think it might be used for a label to an input field, where a user would enter the URL for their companies' FB page.  You can reuse that if you want, as long as the text is consistent with the intended use (that is, a label prompting a user to _enter_ their FB page link could be different from the text used to indicated "here is a link to this companies' FB page", shown to a visitor.

And, yes, please translate the English to Swedish and put that in the `sv.yml` file - then be sure to ask @thesuss to review and correct the Swedish.  We always just start with the Google translate output, and Susanna frequently has to tweak that.

Reply:
@patmbolger Thanks Patrick, glad to be back :) Also, I think I'm just going to create my own keys for my purpose. And yes, I shall ask her, thanks again!